perllib/FixMyStreet/App/Controller/Auth.pm

```diff
@@ -232,6 +232,10 @@ sub process_login : Private {
     $c->detach( '/page_error_403_access_denied', [] )
         if FixMyStreet->config('SIGNUPS_DISABLED') && !$user->in_storage && !$data->{old_user_id};
 
+    # Superusers using 2FA can not log in by code
```

Comment:
There seems to be some inconsistency between this and creating a new report which doesn't allow logging in by code at all for super users. I think I'd be fine with preventing it everywhere.